randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -275,17 +275,21 @@ Extra - asset_id: 6
   * Layers: default
   * Normal Door to Electric Death Room/Door to East Ocean
   > Door to Forgotten Highway Kago Room
-      All of the following:
-          Gravity Suit
-          Hi-Jump Boots or Space Jump
+      Any of the following:
+          All of the following:
+              Gravity Suit
+              Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
```

Comment:
I assume the gravity jumpless WJ is this?
![grafik](https://github.com/randovania/randovania/assets/38186597/3c092482-b44d-4e0b-9871-cc5bf9d20727)

Because that seems to be tight/high at the right

Reply:
Its not as bad as you might think. I can record a video later.

Reply:
https://youtu.be/6tyDDiNLY_o

Reply:
Now that you got a video, might as well add it to the DB :stuck_out_tongue_closed_eyes: